### PR TITLE
HVI Inspection per Vehicle

### PR DESCRIPTION
**Pull request by jrsinnovation**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by jrsinnovation** [comment]
Please validate the connector

File: certified-connectors/shared_hvi-20ins-20specific-20vehicle-5fbc7b7a898d7451-02a9ed7ac5553fca/settings.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (Amjed-Ayoub) [inline certified-connectors/shared_hvi-20ins-20specific-20vehicle-5fbc7b7a898d7451-02a9ed7ac5553fca/settings.json]: @jrsinnovation Please remove the icon.png and settings.json files as they are not required in GitHub.

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]
@jrsinnovation Please rename the connector folder from "shared_hvi-20ins-20specific-20vehicle-5fbc7b7a898d7451-02a9ed7ac5553fca" to the connector name/title "Inspection per vehicle" and would recommend you change the title to "HVI Inspection per vehicle".

Please add a readme file to the pull request. Please include all required sections in the readme file. please see out template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)and for raw code view [here](https://[raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md%5D(https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md)).

File: certified-connectors/shared_hvi-20ins-20specific-20vehicle-5fbc7b7a898d7451-02a9ed7ac5553fca/settings.json (hunk 1 above)
Comment (jrsinnovation) [reply certified-connectors/shared_hvi-20ins-20specific-20vehicle-5fbc7b7a898d7451-02a9ed7ac5553fca/settings.json]: I have deleted those 2 files icon.png and settings.json.

**Comment by jrsinnovation** [comment]
@microsoft-github-policy-service agree company="JRS Innovation"

**Comment by jrsinnovation** [comment]
please review it I have made the changes that you have asked to

**Comment by jrsinnovation** [comment]
Hy Amjed,

please review it i have added a new folder with name HVI Inspection per Vehicle

**Comment by jrsinnovation** [comment]
@Amjed-Ayoub  Please verify it

**Review by jrsinnovation — COMMENTED** [review]
@Amjed-Ayoub  review it

**Comment by jrsinnovation** [comment]
Hi, could you please provide an update. I have to report back the reason for delay in publishing the connector. Let me know if a meeting/ teams meeting can speedup this process?

**Comment by jrsinnovation** [comment]
image

@Amjed-Ayoub you can check i have validated the swagger file then also it is showing failing in git.
Please review it and reply back.

**Comment by jrsinnovation** [comment]
@Amjed-Ayoub  can you please let me know why the validation is failing in git as i have validated it through paconn validate command and there it was successfully validated. Please Check and let me know

paconn validate Image Link:-
https://user-images.githubusercontent.com/130588212/255330457-5aeb133f-d8f1-449b-80ba-9bdd32d142f0.png

**Pull request closed without merging**